Makefile

```diff
@@ -17,7 +17,7 @@ commands :
 
 ## serve            : run a local server.
 serve : lesson-rmd
-	${JEKYLL} serve
+	${JEKYLL} serve -H 0.0.0.0
```

Comment:
Because the ip “127.0.0.1” and “0.0.0.0” is very different, see http://serverfault.com/a/78058.